src/lib/entities/episode/api/get-episode.ts

```diff
@@ -0,0 +1,12 @@
+import { supabaseClient } from '$lib/shared/api';
+import { transformEpisodeRequest, type SBEpisode } from '$lib/shared/api/adapters';
```

Comment:
issue: again, a little violation of FSD, but I understand why. This stuff is not exported from that module

suggestion: let's just export it and import it in a valid way